mito-ai/src/Extensions/AiChat/ChatMessage/AssistantCodeBlock.tsx

```diff
@@ -47,12 +49,13 @@ const AssistantCodeBlock: React.FC<IAssistantCodeBlockProps> = ({
         return (
             <div className={classNames('code-block-container', {
                 'agent-mode': true,
```

Comment:
If this is always true, then we can just write it as classNames('code-block-container', 'agent-mode', { 'agent-mode-collapsed': !isCodeExpanded}